### PR TITLE
[Chore] Bump morley&c

### PR DESCRIPTION
**Pull request by lierdakil**
## Description

Update to new versions of morley &c. ~~This is WIP, I want to see how many network tests I broke.~~

## Related issue(s)

Part of TM-667

#### Related changes (conditional)

- Tests (see [short guidelines](../blob/master/CONTRIBUTING.md#tests))
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - I checked whether I should update the docs and did so if necessary:
    - [ ] [README](../blob/master/README.md)
    - [ ] Haddock
    - [ ] [docs/](../blob/master/docs/)
  - [ ] I updated [changelog](../blob/master/ChangeLog.md) unless I am sure my changes are
        really minor.

#### Stylistic guide (mandatory)

- [x] My commits comply with [the following policy](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
- [x] My code complies with the [style guide](../blob/master/docs/code-style.md).

### CODE REVIEW

**Comment by lierdakil** [comment]
Okay, this is mostly ready, but `stack.yaml` references non-master commits, which should be fixed once the respective upstream MRs are merged.

**Comment by lierdakil** [comment]
I don't think changelog update is necessary, because it would boil down to "updated morley to support Limanet", which arguably isn't informative.

File: src/CLI/L1AddressOrAlias.hs [hunk 1]
```diff
@@ -4,122 +4,16 @@
 {-# OPTIONS_GHC -Wno-orphans #-}
 
 -- TODO: Remove this module when upgrading to the next morley release.
```
Comment (pasqu4le) [inline src/CLI/L1AddressOrAlias.hs:6]: Well, this ended up getting quite small here, why not solve this TODO and actually remove it?

**Review by pasqu4le — APPROVED** [review]

File: src/CLI/L1AddressOrAlias.hs (hunk 1 above)
Comment (lierdakil) [reply src/CLI/L1AddressOrAlias.hs:6]: I was thinking we should probably move this instance upstream (didn't yet create an issue, but I will do that now). Might as well move the instance to some other module, but if we're going to remove it, arguably it wouldn't make a huge difference.

Comment (lierdakil) [reply src/CLI/L1AddressOrAlias.hs:6]: Oh, no, actually I did: https://gitlab.com/morley-framework/morley/-/issues/923, just forgot about it :sweat_smile:

Comment (lierdakil) [reply src/CLI/L1AddressOrAlias.hs:6]: One reason to keep it in a separate module is `-Wno-orphans`. Semantically, it should live either in `Client.Parser` or `CLI.Parser`, but neither of those has `-Wno-orphans` and adding it to a larger module is kind of meh.

Comment (pasqu4le) [reply src/CLI/L1AddressOrAlias.hs:6]: ah, I see, sounds good to me :ok_hand: 

In that case please mention `morley/#923` in this TODO for the next time instead :slightly_smiling_face:

**Pull request merged**